docs/TROUBLESHOOT.md

```diff
@@ -0,0 +1,74 @@
+### Description
+
+Upon running the <docker-compose -f docker-compose.yml -f docker-compose.override.yml up> all the containers are not started mentioned in the docker-compose file. Some containers throw certain errors mentioned below
+
+### To Start/Run the app
+
+◦	docker-compose up -d
+◦	docker-compose stop ibf-dashboard
+◦	cd interfaces/IBF-dashboard
+◦	npm start
+
+### Expected behaviour
+All containers mentioned in the docker-compose file /IBF-system/docker-compose.yml be up and running.
+Application should load and respond properly on the localhost:4200.
+
+### Desktop (please complete the following information):
+
+OS: MAC OS
+Browser : chrome
+Version : 92.*
+
+### Workaround
+    ◦	If the containers ibf-system_ibf-pipeline , ibf-system_ibf-api-service do not start.
+    ◦	If the container do not start automatically via the 'docker compose up' command. The workaround for this would be, manually start the container and run this command.
+
+ docker-compose -f docker-compose.yml -f docker-compose.override.yml up
+
+### After step2 of readme if there is any problem to run the app 
+You can try below steps:
+    ◦	throw away "IBF-app" schema of your local database : drop schema "IBF-app" cascade;
+    ◦	recreate "IBF-app" schema: create schema "IBF-app";
+    ◦	select * from "IBF-app".disaster;
```

Comment:
what is this? just to check if the table exists or something? If so, add that in. Besides, this table cannot exist right after creating the schema. There must be a step in between where the ibf-api-service is restarted (so that the migration-scripts are run). Please add that in.